docs/how-to/write-tasks.md

```
@@ -276,3 +276,83 @@ config:
+
+To refresh to the latest commit:
+
+```bash
+mcpchecker install --update
+```
+
+### Server name mapping
+
+External tasks may reference MCP server names that differ from the names in your local MCP config. During `mcpchecker install`, you are prompted to map each unknown name to a local server:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)